### DISCUSSION TITLE
Is there any solution to use openvino WITHOUT Visual Studio installation?

### DISCUSSION THREAD

**Discussion by RiverLight4** [openvinotoolkit/openvino#26624] [Q&A] [answered]
Hi,
I'd like to use OpenVINO for commercial use, but I'm afraid there is Visual Studio license problem. Could you please help me?

### Background
I'd like to do inference with OpenVINO on edge computer product; e.g. System-on-Board mini PC.
However, it seems that Visual Studio installation is needed when using OpenVINO now.
Because the company that I work for is included with "Enterprise" defined by Visual Studio community license, I'm afraid that license issue would be raised in the future, if I would release the product with OpenVINO models, including Visual Studio.

I think there are some solutions:
- Buy Visual Studio licenses for each product (too expensive)
- Install Visual Studio Community Edition by the product user (too difficult)
- Use some Visual Studio runtime, without installing Visual Studio

### Question
- Is there any solution to run OpenVINO without Visual Studio?
- Or, is there any matched license of Visual Studio?
- What EXE/DLL are needed to run model with OpenVINO?

I think I should ask to Microsoft about detail Visual Studio license, but I’d like to make it clear why Visual Studio is needed when running OpenVINO before I ask.

**Comment by ilya-lavrenov** [comment, marked as answer]
You don't need Visual Studio, you need Microsoft Visual C++ Redistributable https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170
It's required for almost ALL Windows apps

**Reply by RiverLight4** [reply]
Thank you very much.
As you mentioned, I installed Microsoft Visual C++ Redistributable `vc_redist_x64.exe` then OpenVINO is worked.

Because there is no description in [Installation manual](https://docs.openvino.ai/2024/get-started/install-openvino/install-openvino-archive-windows.html) about this, I hope that this answer will be written in the future.
